### PR TITLE
js loops completed

### PR DESCRIPTION
**Pull request by dcatan89**
Screen Shot 2022-02-02 at 3 14 36 PM

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ */
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  while (currentNumber < 10) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
+  return numbers;
+}
```
Comment (TimelikeClosure) [inline javascript-loops/main.js:17]: The returned array should include `10`:

---

image

---

image

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 2;
+  while (currentNumber < 20) {
+    evenNumbers.push(currentNumber);
+    currentNumber += 2;
+  }
+  return evenNumbers;
+}
```
Comment (TimelikeClosure) [inline javascript-loops/main.js:27]: ... and this one should include `20`

File: javascript-loops/main.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+function repeatWord(word, times) {
+  var count = 1;
+  var repeated = '';
+  while (count < times) {
+    repeated += word;
+    count++;
+  }
+  return repeated;
+}
```
Comment (TimelikeClosure) [inline javascript-loops/main.js:37]: `repeatWord` should include `times` copies of `word`, not `times - 1` copies:

---

image

---

image

File: javascript-loops/main.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+function doubleAll(numbers) {
+  var doubled = [];
+  for (var i = 0; i < numbers.length; i++) {
+    numbers[i] = numbers[i] * 2;
+    doubled.push(numbers[i]);
+  }
+  return doubled;
+}
```
Comment (TimelikeClosure) [inline javascript-loops/main.js:52]: `doubleAll` should not modify the contents of the `numbers` array:

---

image
****

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by dcatan89** [comment]
Screen Shot 2022-02-02 at 3 31 16 PM

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**